bin/GameIndex.dbf

```diff
@@ -40213,6 +40242,12 @@ Name   = Metal Gear Solid 3 - Subsistence [Disc1of3]
 Region = NTSC-U
 Compat = 5
 ---------------------------------------------
+Serial = SLUS-21243
```

Comment:
This shouldn't be moved, the serials are in order numerically, not alphabetically or grouped by game.